usr.bin/procstat/procstat_auxv.c

```diff
@@ -64,16 +64,16 @@ procstat_auxv(struct procstat *procstat, struct kinfo_proc *kipp)
 	static char prefix[256];
 
 	if ((procstat_opts & PS_OPT_NOHEADER) == 0)
-		xo_emit("{T:/%5s %-16s %-16s %-16s}\n", "PID", "COMM", "AUXV",
+		xo_emit("{T:/%5s %-19s %-16s %-16s}\n", "PID", "COMM", "AUXV",
 		    "VALUE");
 
 	auxv = procstat_getauxv(procstat, kipp, &count);
 	if (auxv == NULL)
 		return;
-        snprintf(prefix, sizeof(prefix), "%5d %-16s", kipp->ki_pid,
-            kipp->ki_comm);
+	snprintf(prefix, sizeof(prefix), "%5d %-19s", kipp->ki_pid,
+	    kipp->ki_comm);
 
-	xo_emit("{e:process_id/%5d/%d}{e:command/%-16s/%s}", kipp->ki_pid,
+	xo_emit("{e:process_id/%5d/%d}{e:command/%-19s/%s}", kipp->ki_pid,
```

Comment:
Since MAXCOMLEN doesn't include the nul terminator (the kernel is full of `MAXCOMLEN+1` I suppose for this reason), shouldn't the width be 20?

Reply:
This is the printable width so should be 19. With a too-long command name it aligns correctly:
```
root@cheribsd-morello-purecap:~ # ps
 PID TT  STAT    TIME COMMAND
1004 u0  Is   0:00.84 login [pam] (login)
1083 u0  S    0:01.42 -sh (sh)
1294 u0  T    0:00.65 cheribsdtest-hybrid -a
1360 u0  T    0:01.19 cheribsdtest-hybrid-dynamic-mt -a (cheribsdtest-hybrid)
1410 u0  R+   0:00.15 ps
root@cheribsd-morello-purecap:~ # procstat auxv -v 1360
  PID COMM                AUXV             VALUE           
 1360 cheribsdtest-hybrid AT_PHDR          0x400328720040
 1360 cheribsdtest-hybrid AT_PHENT         56
 1360 cheribsdtest-hybrid AT_PHNUM         12
 1360 cheribsdtest-hybrid AT_PAGESZ        4096
 1360 cheribsdtest-hybrid AT_FLAGS         0
 1360 cheribsdtest-hybrid AT_ENTRY         0x4003287410ec
 1360 cheribsdtest-hybrid AT_BASE          0x5920ed55b000
...
```